scaphandre/CHANGELOG.md

```diff
@@ -0,0 +1,8 @@
+# CHANGELOG - Scaphandre
+
+## 1.0.0 / 2024-05-16
```

Comment:
This should be managed by towncrier. Out of curiosity, which `ddev create` command did you run when starting to work on this integration?